reactive/hacluster.py

```diff
@@ -0,0 +1,41 @@
+from charms.reactive import when, when_not, clear_flag, set_flag
+from charms.reactive import endpoint_from_flag
+from charmhelpers.core import hookenv
+from charms.layer.kubernetes_common import get_ingress_address
+from charms.layer.hacluster import services
+
+
+@when('ha.connected', 'layer.hacluster.services_configured')
+@when_not('layer.hacluster.configured')
+def configure_hacluster():
+    """Configure HA resources in corosync"""
+    hacluster = endpoint_from_flag('ha.connected')
+    vips = hookenv.config('ha-cluster-vip').split()
+    dns_record = hookenv.config('ha-cluster-dns')
+    if vips and dns_record:
+        set_flag('layer-hacluster.dns_vip.invalid')
+        msg = "Unsupported configuration. " \
+              "ha-cluster-vip and ha-cluster-dns cannot both be set",
+        hookenv.log(msg)
+        return
+    else:
+        clear_flag('layer-hacluster.dns_vip.invalid')
+    if vips:
+        for vip in vips:
+            hacluster.add_vip(hookenv.application_name(), vip)
```

Comment:
What if the operator changes `ha-cluster-vip` after deployment? I see vips added to the interface here, but when are old ones removed?

Reply:
I have a question in to @javacruft about that. I don't see any way to remove a vip from via the interface outside of deleting a resource, which seems a little how the sausage is made.

Reply:
I've added https://github.com/openstack/charm-interface-hacluster/pull/1 to address deletions